common/playlist.c

```diff
@@ -170,9 +170,10 @@ void playlist_shuffle(struct playlist *pl)
 {
     for (int n = 0; n < pl->num_entries; n++)
         pl->entries[n]->original_index = n;
+    mp_rand_state s = mp_rand_seed(0);
```

Comment:
Could you add early exit when `pl->num_entries == 1`? Everything is playlist in mpv, no point in generating state in this case.

Reply:
> no point in generating state in this case.

Went a different route: aa2d54ebb47db8a2db582a6a45a81afecd27ba89. What do you think?